bluefloodserver/collect.py

```diff
@@ -25,18 +27,25 @@ def flush(self, metrics):
 
 class BluefloodFlush(IFlush):
 
-    def __init__(self, client, ttl=60*60*24):
+    def __init__(self, client, ttl=60*60*24, metric_prefix=None):
         self.client = client
         self.ttl = ttl
+        self.metric_prefix = metric_prefix
+        log.msg('Prepending all metrics with custom string {}'.format(metric_prefix), level=logging.DEBUG)
 
     @inlineCallbacks
     def flush(self, metrics):
         for name, time, value in metrics:
+            if self.metric_prefix != "":
```

Comment:
You can do "if not self.metric_prefix". I dont know if it is the python way of doing it but I had seen code like that.